apps/aehttp/priv/swagger.json

```diff
@@ -1623,6 +1687,42 @@
         }
       }
     },
+    "Contract" : {
+      "type" : "object",
+      "properties" : {
+        "code" : {
+          "type" : "string"
+        },
+        "options" : {
+          "type" : "string"
+        }
+      }
+    },
+    "ByteCode" : {
+      "type" : "object"
+    },
+    "ContractCall" : {
+      "type" : "object",
+      "properties" : {
+        "code" : {
+          "type" : "string"
+        },
+        "function" : {
+          "type" : "string"
+        },
+        "arg" : {
```

Comment:
Is the idea to extend this later to a list of argument objects, of which the contract defines the type, or is this always a string representation that is parsed lower down? If so, it might be nice with a comment in YAML where the string representation is defined.

Reply:
The function is just the name of the called function.